dali/aux/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_subdirectory(optical_flow)
+
+if (BUILD_TEST)
```

Comment:
I think it can be removed

Reply:
As far as I understand, `DALI_TEST_SRCS` also has to be propagated to parent scope. I'll remove globbing, since it can be removed, but it seems, that `set(DALI_TEST_SRCS ${DALI_TEST_SRCS} ${tmp} PARENT_SCOPE)` has to stay, as well as the `if ` here

Reply:
Right, I forgot about nested tests.
I think you can get rid of this `if` here anyway.